### PR TITLE
add script lel

### PR DESCRIPTION
**Pull request by baptiste-martin-campus**
it's a great script

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
-    <main></main>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>CDA 2025</title>
+</head>
+
```
Comment (AnaisBonin) [inline index.html:9]: remove line

File: index.html [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
+<body>
+  <header>
+    <h1>Helloooooo</h1>
+  </header>
+  <main></main>
+  <script>
+    var coucou = document.getElementsByClassName("title");
+  </script>
+</body>
+
```
Comment (AnaisBonin) [inline index.html:19]: remove line

File: index.html (hunk 1 above)
Comment (baptiste-martin-campus) [reply index.html:9]: no

File: index.html (hunk 2 above)
Comment (baptiste-martin-campus) [reply index.html:19]: non plus

**Review by seb-lecampus — APPROVED** [review]

**Review by AnaisBonin — APPROVED** [review]
Fine.

**Pull request merged**